ui/js/controllers/jobs.js

```diff
@@ -71,6 +71,10 @@ treeherderApp.controller('JobsCtrl', [
             }
         };
 
+        if($location.search().revision === 'undefined'){
+            thNotify.send("Revision Undefined", "danger");
```

Comment:
Optional, but another style nuance, I've started doing this lately with thNotify, and our codebase in general.
```
("Customer facing string", 'danger');
```
Where strings the user sees receive double quotes " ", and internal-use strings generally receive single quotes ' '.